src/modules/auth/ToUAnimation.json

```diff
@@ -1 +1,1479 @@
":0,"nm":"Bee precomp 2_wingspan15_regularspeed_v2","refId":"comp_0","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":226,"ix":10},"p":{"a":1,"k":[{"i":{"x":0.479,"y":0},"o":{"x":0.333,"y":0},"t":166,"s":[499.007,414.968,0],"to":[-15.243,14.718,0],"ti":[-21.259,-17.918,0]},{"i":{"x":0.833,"y":0.687},"o":{"x":0.399,"y":0.217},"t":183,"s":[493.009,480.918,0],"to":[21.259,17.918,0],"ti":[-34.483,38.54,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":191.742,"s":[614.633,477.588,0],"to":[34.483,-38.54,0],"ti":[-54.238,58.155,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":200.12,"s":[740.421,303.231,0],"to":[50.579,-54.231,0],"ti":[-202.84,0,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":211.037,"s":[1077.193,189.669,0],"to":[192.807,0,0],"ti":[-143.219,-6.89,0]},{"t":222,"s":[1453.219,288.89,0]}],"ix":2,"x":"var $bm_rt;\n$bm_rt = transform.position;"},"a":{"a":0,"k":[50,50,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":1,"w":100,"h":100,"ip":107,"op":166,"st":0,"bm":0},{"ddd":0,"ind":4,"ty":0,"nm":"Bee precomp 2_wingspan15_regularspeed_v2","refId":"comp_0","sr":1,"ks":{"o":{"a":0,"k":100,"ix":11},"r":{"a":0,"k":90,"ix":10},"p":{"a":1,"k":[{"i":{"x":0.833,"y":0.885},"o":{"x":0.167,"y":0.119},"t":0,"s":[-58,813,0],"to":[101.021,6.254,0],"ti":[-107.65,44.052,0]},{"i":{"x":0.294,"y":0.538},"o":{"x":0.167,"y":0.22},"t":12.427,"s":[328.23,776.977,0],"to":[99.431,-46.406,0],"ti":[123.994,1.845,0]},{"i":{"x":0.833,"y":0.862},"o":{"x":0.522,"y":0.322},"t":31,"s":[260,562,0],"to":[-90,1,0],"ti":[-214.465,-66.519,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":52.105,"s":[330,814,0],"to":[187,58,0],"ti":[-14.997,79.112,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":64.351,"s":[661.173,637.258,0],"to":[25.756,-135.867,0],"ti":[40.568,20.284,0]},{"i":{"x":0.223,"y":1},"o":{"x":0.167,"y":0.754},"t":79.038,"s":[590.207,413.821,0],"to":[-40.568,-20.284,0],"ti":[15.243,-14.718,0]},{"i":{"x":0.833,"y":1},"o":{"x":0.886,"y":0},"t":109.137,"s":[499.007,414.968,0],"to":[-15.243,14.718,0],"ti":[15.243,-14.718,0]},{"i":{"x":0.505,"y":0},"o":{"x":0.333,"y":0},"t":165.132,"s":[499.007,414.968,0],"to":[-15.243,14.718,0],"ti":[-21.259,-17.918,0]},{"i":{"x":0.833,"y":0.687},"o":{"x":0.399,"y":0.217},"t":183,"s":[493.009,480.918,0],"to":[21.259,17.918,0],"ti":[-34.483,38.54,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":191.742,"s":[614.633,477.588,0],"to":[34.483,-38.54,0],"ti":[-54.238,58.155,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":200.12,"s":[740.421,303.231,0],"to":[50.579,-54.231,0],"ti":[-202.84,0,0]},{"i":{"x":0.833,"y":0.833},"o":{"x":0.167,"y":0.167},"t":211.037,"s":[1077.193,189.669,0],"to":[192.807,0,0],"ti":[-143.219,-6.89,0]},{"t":222,"s":[1453.219,288.89,0]}],"ix":2,"x":"var $bm_rt;\n$bm_rt = transform.position;"},"a":{"a":0,"k":[50,50,0],"ix":1},"s":{"a":0,"k":[100,100,100],"ix":6}},"ao":1,"w":100,"h":100,"ip":0,"op":107,"st":0,"bm":0}],"markers":[]}
```

Comment:
We don't want this formatted. If the linter did it automatically, then I guess let's make an exception for json files in general to not format.